docker/daemon/daemon_src.go

```diff
@@ -9,9 +9,18 @@ import (
 	"github.com/containers/image/types"
 	"github.com/pkg/errors"
 	"golang.org/x/net/context"
+	"runtime"
 )
 
-const temporaryDirectoryForBigFiles = "/var/tmp" // Do not use the system default of os.TempDir(), usually /tmp, because with systemd it could be a tmpfs.
+var temporaryDirectoryForBigFiles string
+
+func init() {
```

Comment:
Instead of doing the same thing in multiple places, I was wondering whether there should be a shared util function _GetTemporaryDirectoryForBigFiles_ somewhere. AFAICT, there is no obvious location yet for these type of utility method. Should I create one? And if so where?

Reply:
Yes, please, move this into a single implementation.

Probably a subpackage in the `internal` hierarchy, something like `containers/image/internal/tmpdir`.

I wouldn’t be opposed to making this a public subpackage under `containers/image/pkg/` either, it could be useful for other software; OTOH having such a containers-independent utility under `containers/image` feels a little weird.

Reply:
Ok. I think I go for the internal/tmpdir